src/MovieContainer/MovieContainer.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import MovieContainer from './MovieContainer'
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+describe('MovieContainer', () => {
+  it('should display all movieCards in the allMovies array', () => {
+    const mockUser = {};
+    const mockAllMovies = [
+      {
+        id: 1,
+        posterPath: "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+        title: "Money Plane",
+        averageRating: 4
+      },
+      {
+        id: 2,
+        posterPath: "https://image.tmdb.org/t/p/original//aKx1ARwG55zZ0GpRvU2WrGrCG9o.jpg",
+        title: "Mulan",
+        averageRating: 4
+      },
+      {
+        id: 3,
+        posterPath: "https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg",
+        title: "Rogue",
+        averageRating: 5
+      }
+    ];
+    const mockUserRatings = [];
+    const mockDetermineShowPageButton = jest.fn();
+    const mockRetrieveAllRatings = jest.fn();
+
+    render(
+      <MovieContainer
+        user={mockUser}
+        allMovies={mockAllMovies}
+        determineShowPageButton={mockDetermineShowPageButton}
+        userRatings={mockUserRatings}
+        retrieveAllRatings={mockRetrieveAllRatings}
+      />
+    )
+
+
+    const movieOne = screen.getByText("Money Plane");
```

Comment:
We should definitely refactor this so that we are testing with `async / await`, that way we are mocking the resolved value from the fetch request that returns all of the movies from the API.